impl/src/test/java/org/jboss/arquillian/graphene/enricher/AbstractGrapheneEnricherTest.java

```diff
@@ -42,13 +42,13 @@
 import org.junit.rules.ExpectedException;
 import org.junit.runner.RunWith;
 import org.mockito.Mock;
-import org.mockito.runners.MockitoJUnitRunner;
+import org.mockito.junit.MockitoJUnitRunner;
 
 /**
  * @author <a href="mailto:jhuska@redhat.com">Juraj Huska</a>
  * @author <a href="mailto:jpapouse@redhat.com">Jan Papousek</a>
  */
-@RunWith(MockitoJUnitRunner.class)
```

Comment:
I understand it's porting to a newer version of Mockito, but does it fail due to unused stubs? If so maybe it's worth opening an issue and cleaning this debt.

Reply:
yes, it fails due to unused stubs. Always did.

Reply:
Issue created: https://issues.redhat.com/browse/ARQ-2229